server/plugin/subscriptions.go

```diff
@@ -15,24 +16,39 @@ import (
 const (
 	SubscriptionsKey     = "subscriptions"
 	excludeOrgMemberFlag = "exclude-org-member"
+	renderStyleFlag      = "render-style"
 )
 
 type SubscriptionFlags struct {
 	ExcludeOrgMembers bool
+	RenderStyle       string
 }
 
-func (s *SubscriptionFlags) AddFlag(flag string) {
+func (s *SubscriptionFlags) AddFlag(flag string, value string) error {
 	switch flag { // nolint:gocritic // It's expected that more flags get added.
```

Comment:
I think we can get rid of this linter exclusion here now

```suggestion
	switch flag {
```